devtron-labs/dashboard
# feat: Tags new UI + propagation to resource

# Description

This PR has the new UI changes of app creation and app overview around the tags

Fixes # [(AB#1733)](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/1733)

## Type of change


# How Has This Been Tested?
- Created new app and verified the new UI
- Open existing app go in App overview section
- Edit the tags and verify new UI


# Checklist:
